docs/release-notes/snapcraft-8-9.rst

```diff
@@ -172,3 +173,4 @@ and :literalref:`@tigarmo<https://github.com/tigarmo>`
 .. _cargo registry: https://doc.rust-lang.org/cargo/reference/registries.html
 .. _#5107: https://github.com/canonical/snapcraft/pull/5107
 .. _#5272: https://github.com/canonical/snapcraft/pull/5272
+.. _#5491: https://github.com/canonical/snapcraft/pull/5491
```

Comment:
It is curious that we use a "pull" link to reference issues. Is that on purpose?

Reply:
Oops, that was a copy/paste mistake.

Reply:
Fixed.